### DISCUSSION TITLE
有联系方式吗？

### DISCUSSION THREAD

**Discussion by baichengzhou** [NXY666/JsjiamiV6-Decryptor#1] [Q&A] [answered]
微信或者电话联系方式有吗？

**Comment by baichengzhou** [comment]
您这么直接针对我们，最近正在解决淘宝和闲鱼上的侵权诉讼，想+一下你一起诉讼一下。

**Reply by AmamiyaRain** [reply]
🤣你急啦？

**Reply by baichengzhou** [reply]
我从来不急，我只是提醒下。

**Comment by NXY666** [comment, marked as answer]
任何关于本项目的交流均在github上公开进行，有任何相关的疑问和建议可以提交issue或者discussion。

**Reply by baichengzhou** [reply]
你直接针对我们的产品破解，我们是有软著的，github不是法外之地。你可以一直这么坚定。

**Comment by baichengzhou** [comment]
过些天再来找你把。到时候希望你一直坚定下去
